cloudformation/application-setup-basic/stacks/application/application.yml

```diff
@@ -0,0 +1,50 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: Combines all stacks necessary to deploy an application.
+
+Parameters:
+
+  RegistryStackName:
+    Type: String
+    Description: The name of the ECR registry stack.
+
+  # network stack parameters
+  NetworkStackTemplateUrl:
+    Type: String
+    Description: URL to the network stack template in an S3 bucket.
+
+  # Service stack parameters
+  ServiceStackTemplateUrl:
+    Type: String
+    Description: URL to the service stack template in an S3 bucket.
+  ServiceStackImageUrl:
+    Type: String
+    Description: URL to the Docker image
+  ServiceStackUserPoolClientSecret:
+    Type: String
+    Description: The client secret to access the Cognito UserPool.
+
+Resources:
+
+  NetworkStack:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Ref 'NetworkStackTemplateUrl'
+
+  ServiceStack:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: !Ref 'ServiceStackTemplateUrl'
+      Parameters:
+        NetworkStackName: !GetAtt NetworkStack.Outputs.StackName
+        RegistryStackName: !Ref 'RegistryStackName'
```

Comment:
I moved the `ECSTaskDefinitionExecutionRole` from the network stack to the registry stack to solve the bootstrapping problem. Here, we pass the registry stack name so that the service stack can import the role from the registry stack.

I did this to solve the bootstrapping problem. Now, we have a clean, repeatable order: first start the registry stack, then start the application stack.